python/pynessie/nessie_client.py

```diff
@@ -51,21 +38,24 @@ class NessieClient(object):
 
     def __init__(self: "NessieClient", config: confuse.Configuration) -> None:
         """Create a Nessie Client from known config."""
-        self._base_url = config["endpoint"].get()
-        self._ssl_verify = config["verify"].get(bool)
+        self._endpoints = EndPoints(config)
         self._commit_id: str = cast(str, None)
 
         try:
             self._base_branch = config["default_branch"].get()
         except confuse.exceptions.NotFoundError:
             self._base_branch = None
 
+    def base_url(self: "NessieClient") -> str:
```

Comment:
if we want to expose this as a method I would prefer as a `property`

Reply:
I rolled back this changes in cce999859b660e462b538b51b451f438e210ee6e